sips/sip-021/sip-021-nakamoto.md

```diff
@@ -0,0 +1,1051 @@
+prior to selection.**
+1. **Use stackers to police miner behavior.**  A cryptographic sortition causes
+the Stackers to carry out a _tenure change_ by (a) agreeing on a "last-signed"
+block from the current miner, and (b) agreeing to only sign blocks from the new
+miner which descend from this last-signed block. Thus, Stackers police miner
+behavior --  Stackers prevent miners from mining forks during their tenure, and
+ensure that they begin their tenures by building atop the canonical chain tip.
+This **further prevents miners from forking the stacks blockchain.**
+1. **Require Stacks miners to commit the _indexed block hash_ of the first block
+produced by the last Stacks miner in their block-commit transactions on
+Bitcoin.** This is the SHA512/256 hash of both the _consensus hash_ of all
+previously-accepted burnchain transactions that Stacks recognizes, as well as
+the hash of the block itself (a `block-commit` today only contains the hash of
+the Stacks block).  This will anchor the Stacks chain history to Bitcoin up to
+the start of the previous miner's tenure, _as well as_ all causally-dependent
+Bitcoin state that Stacks has processed. This **ensures Bitcoin finality**,
+**resolves miner connectivity issues** by putting fork prevention on stackers,
+and allows nodes with up-to-date copies of the Stacks chain state to **identify
+which Stacks blocks are affected by a Bitcoin reorg** and recover the affected
+Stacks transactions.
+1. **Stackers identify the sequence of orphaned transactions after a fork for
+transaction replay.** This can be implemented after this SIP is ratified because
+it is not consensus-critical, and can be deployed incrementally because it only
+requires sufficient numbers of Stackers to cooperate.
+1. **Adopt a Bitcoin MEV solution which punishes block-commit censorship.** The
+probability should be altered such that:
+    1. The probability of winning is non-zero only if the miner attempted to
+mine in the majority of the past ten cryptographic sortitions. This is not true
+today -- if a miner is the sole miner in a Bitcoin block, then it has 100%
+chance of winning. In this system, it can be the case that no miner has a
+non-zero chance of winning. A Bitcoin MEV miner who spends a _de minimis_ PoX
+payout in Bitcoin blocks it wins, but does not compete in other blocks, will
+_never_ win sortition unless the Bitcoin miner also has the majority of Bitcoin
+mining power.
+    1. The probability of winning is inversely proportional to the larger of
+either the system's total commits in the current block, or the system's median
+total commits over the past ten blocks.  This means that a sudden decrease in
+the Bitcoin block's total PoX payout, such as due to the exclusion of other
+Stacks miners' `block-commit`s by a Bitcoin MEV miner, does _not_ increase the
+likelihood that the MEV miner's _de minimis_ `block-commit` wins.
+    1. The probability of winning is absolute, not relative. The sum of each
```

Comment:
there is an issue with the numbers here. Both are 1. and 1. instead of 1. and 2. for these bullet points.